pywps/exceptions.py

```diff
@@ -66,14 +65,14 @@ def get_response(self, environ=None):
             'name': escape(self.name),
             'description': self.get_description(environ)
         }
-        doc = text_type((
-            u'<?xml version="1.0" encoding="UTF-8"?>\n'
-            u'<!-- PyWPS {version} -->\n'
-            u'<ows:ExceptionReport xmlns:ows="http://www.opengis.net/ows/1.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.opengis.net/ows/1.1 http://schemas.opengis.net/ows/1.1.0/owsExceptionReport.xsd" version="1.0.0">\n'  # noqa
-            u'  <ows:Exception exceptionCode="{name}" locator="{locator}" >\n'
-            u'      {description}\n'
-            u'  </ows:Exception>\n'
-            u'</ows:ExceptionReport>'
+        doc = str((
+            '<?xml version="1.0" encoding="UTF-8"?>\n'
+            '<!-- PyWPS {version} -->\n'
+            '<ows:ExceptionReport xmlns:ows="http://www.opengis.net/ows/1.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.opengis.net/ows/1.1 http://schemas.opengis.net/ows/1.1.0/owsExceptionReport.xsd" version="1.0.0">\n'  # noqa
```

Comment:
Shouldn't `version="1.0.0"` either be the actual WPS version, or correspond to `1.1.0` OWS schema?